gammapy/estimators/map/asmooth.py

```diff
@@ -42,6 +42,8 @@ class ASmoothMapEstimator(Estimator):
         Significance estimation method.
     threshold : float
         Significance threshold.
+    energy_edges : `~astropy.units.Quantity`
+        Energy edges of the target maps bins.
```

Comment:
in this docstring, the edges is only 1 quantity, but in fact this is a list with [emin, emax]
Also, one of the user's issue is that the user's edge can not used, but rather the closest bins of the dataset map. This is the tricky point to explain

Reply:
Yes for sure. 
@cgalelli and I are working on that together, our thought was to explain in the documentation what the `energy_edges` is, and then link to that in the doc string. 
We wanted to start a PR so we could see how committing to the same PR works.